README.md

```diff
@@ -3,42 +3,46 @@
 Performance measurement app for KUKSA databroker.
 
 ```
-[00:00:00]  [========================================================================================================]    1000/1000    iterations
-
-Summary:
-  API: SdvDatabrokerV1
-  Elapsed time: 0.16 s
-  Rate limit: None
-  Sent: 1000 iterations * 1 signals = 1000 updates
-  Skipped: 10 updates
-  Received: 990 updates
-  Fastest:   0.042 ms
-  Slowest:   0.686 ms
-  Average:   0.130 ms
+[00:00:10] Group: Frame 1 | Cycle(ms): 10 | Current latency: 0.519 ms [====================================================================]      6/6      seconds
+[00:00:09] Group: Frame 2 | Cycle(ms): 20 | Current latency: 0.562 ms [====================================================================]      6/6      seconds
+[00:00:09] Group: Frame 3 | Cycle(ms): 30 | Current latency: 0.732 ms [====================================================================]      6/6      seconds
+
+Global Summary:
+  API: KuksaValV2
+  Total elapsed seconds: 6
+  Skipped test seconds: 2
+  Total signals: 8 signals
+  Sent: 2283 signal updates
+  Skipped: 768 signal updates
+  Received: 1515 signal updates
+  Signal/Second: 378 signal/s
```

Comment:
Repeating signal/second and signal/s..
`Throughput: ??? signal updats / s` perhaps?